libraries/AP_DDS/README.md

```diff
@@ -140,35 +141,43 @@ ros2 run micro_ros_agent micro_ros_agent serial -b 115200 -D /dev/pts/2  -r dds_
 
 ### Using the ROS2 CLI to Read Ardupilot Data
 
-If you have installed the microROS agent and ROS-2 Humble
+If you have installed the microROS agent and ROS2 Humble
 
 - Source the ros2 installation
-  - ```source /opt/ros/humble/setup.bash```
-
-- If SITL is running alongise MicroROS Agent, you should be able to see the agent here and view the data output.
-
-
-```
-$ ros2 node list
-/Ardupilot_DDS_XRCE_Client
-
-$ ros2 topic list  -v
-Published topics:
- * /ROS2_Time [builtin_interfaces/msg/Time] 1 publisher
- * /parameter_events [rcl_interfaces/msg/ParameterEvent] 1 publisher
- * /rosout [rcl_interfaces/msg/Log] 1 publisher
-
-Subscribed topics:
-
-$ ros2 topic hz /ROS2_Time
-average rate: 50.115
-        min: 0.012s max: 0.024s std dev: 0.00328s window: 52
-
-$ ros2 topic echo /ROS2_Time 
-sec: 1678668735
-nanosec: 729410000
----
-```
+  ```bash
+  source /opt/ros/humble/setup.bash
+  ```
+- Run the microROS agent
+  ```bash
+  cd ardupilot/libraries/AP_DDS
+  ros2 run micro_ros_agent micro_ros_agent serial -b 115200 -D /dev/pts/2  -r dds_xrce_profile.xml # (assuming we are using tty/pts/2 for Ardupilot)
```

Comment:
I'm a bit conflicted about adding the same commands as earlier down here. Perhaps the headers need to be restructured, but the point is the whole README is done in order, and everything stays running. Is there a reason we need to duplicate the stuff here? The reason I am worreid about that is a user assuming they need to leave everything running, then running two conflicting micro ros agents or two SITL's.

Reply:
The problem is that if the whole README is done in order Ardupilot SITL will run before the microROS agent, causing the DDS Client to fail initialization. I agree that a header overhaul might be the best way to solve this

Reply:
The bootup order is very temporary. It's my next action item to make it boot up regardless of order and robust to restarts on either side. It won't be a thing very long at all.

Reply:
By separating SETUPS from actually running the example we can assure that no command runs twice